src/components/VSkipToContentContainer.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div ref="containerNode">
+    <slot />
+    <VTeleport to="skip-to-content">
+      <VButton
+        class="focus:fixed focus:absolute ms-2 mt-2 z-50"
+        :class="$style.skipButton"
+        @click="skipToContent"
+        >{{ $t('skip-to-content') }}</VButton
+      >
+    </VTeleport>
+  </div>
+</template>
+
+<script>
+import { defineComponent, ref } from '@nuxtjs/composition-api'
+
+import { getFirstTabbableIn } from 'reakit-utils/tabbable'
+import { ensureFocus } from 'reakit-utils/ensureFocus'
+
+import VTeleport from '~/components/VTeleport/VTeleport.vue'
+import VButton from '~/components/VButton.vue'
+
+/**
+ * Demarcates the section that the "skip to content"
+ * keyboard accessibility button should skip to.
+ *
+ * It also enables the button to appear on the page. If
+ * this component isn't rendered on the page then the
+ * button will not render.
+ */
+export default defineComponent({
+  name: 'VSkipToContentContainer',
+  components: { VTeleport, VButton },
+  props: {
+    initialFocusNode: {
+      type: process.server ? Object : HTMLElement,
+      required: false,
+    },
+  },
+  setup(props) {
+    const containerNode = ref()
+
+    const skipToContent = () => {
+      if (!(containerNode.value || props.initialFocusNode)) return
+      const tabbable =
+        props.initialFocusNode || getFirstTabbableIn(containerNode.value, true)
+      ensureFocus(tabbable)
+    }
+
+    return { containerNode, skipToContent }
+  },
+})
+</script>
+
+<style module>
+.skipButton:not(:focus) {
+  @apply sr-only;
+}
```

Comment:
We want to remove the `sr-only` class when the button is focused so that it becomes visible for sighted keyboard users who are not using a screen reader. This seems like the easiest way to do that but maybe there's something built into Tailwind that would simplify it?

Reply:
I think this is ideal, although it would be cool (confusing perhaps) if tailwind had a `not` selector.

Reply:
FWIW I did try using `sr-only focus:not-sr-only` but `not-sr-only` doesn't remove all the `sr-only` styles, just the ones preventing the element from being visible, meaning the button margins and padding are all messed up. Unfortunately it seems like this really is the best approach for `not-focus:sr-only` 😭